doc/setting-up-postgres.md

```diff
@@ -0,0 +1,95 @@
+# Postgres guide
+
+Here is a quick run down on how to set up postgres
+
+Feel free to skip, laugh or feedback if you actually know already how to postgres.
+
+I would recommend using the system postgres for your OS as it has proven to be relatively stable once set up. (not the docker route. it seems easier but you can get problems if you have a mismatched server / client / dev libraries then you'll have to sort those out on your own)
+
+1. Install postgres and postgres development libraries for your platform
+   Based on your OS
+
+- linux
+  - [fedora](https://docs.fedoraproject.org/en-US/quick-docs/postgresql/)
+  - [ubuntu/debian](https://linuxize.com/post/how-to-install-postgresql-on-debian-10/)
+- mac os
+  - don't use the app!
```

Comment:
```suggestion
  - Postgres.app (https://postgresapp.com/) If you use this, be aware, it can cause problems when synchronising versions between the app postgres, the client postgres and the build dependency postgres.
```